src/js/components/LocalImplementationProjects.jsx

```diff
@@ -0,0 +1,31 @@
+import useAirtableCMS from "../hooks/useAirtableCMS";
+import Markdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+const LocalImplementationProjects = (props) => {
+  const lip = useAirtableCMS({
+    baseID: "app1YqNgXXkVH04nO",
+    tableName: "Local Implementation Projects",
+    keyField: "name",
+    fieldMapping: {
+      name: "Name",
+      desc: "Description",
+      notes: "Notes"
+    },
+  });
+
+  return (
+    <div className="component LocalImplementationProjects">
+      {lip.map((record) => {
+        return (
+          <div className="container">
```

Comment:
Elements in an iterator should have a "key" prop for efficient diffing by React (see https://stackoverflow.com/questions/28329382/understanding-unique-keys-for-array-children-in-react-js)